config/etc/system-image/writable-paths

```diff
@@ -82,9 +67,8 @@
 # dbus bus policy
 /etc/dbus-1/system.d                    auto                    persistent  transition  none
 /etc/modprobe.d                         auto                    synced      none        none
+# FIXME: do we still need ppp on the image?
```

Comment:
Yes we do. We use ppp from the core snap in the modem-manager snap. However need to check if we need /etc/ppp writable or not.